multi-node/aws/pkg/config/tls_config.go

```diff
@@ -170,15 +174,32 @@ func compressData(d []byte) (string, error) {
 	return base64.StdEncoding.EncodeToString(buff.Bytes()), nil
 }
 
-func (r *RawTLSAssets) Compact() (*CompactTLSAssets, error) {
+func (r *RawTLSAssets) Compact(cfg *Config) (*CompactTLSAssets, error) {
+
+	awsConfig := aws.NewConfig()
+	awsConfig = awsConfig.WithRegion(cfg.Region)
+	kmsSvc := kms.New(session.New(awsConfig))
+
 	var err error
 	compact := func(data []byte) string {
 		if err != nil {
 			return ""
 		}
+
+		if cfg.KMSKeyARN != "" {
```

Comment:
This is a hack so that we can still assemble/validate the cloud-config/stack template files if there are not aws credentials available. For unit testing without aws api access.